### PR TITLE
Customize portfolio with fun facts about myself and an image

### PR DESCRIPTION
**Pull request by linamontes10-zz**
Basic customization of the week 1 sprint project, a portfolio website. Adding an image, changing the script to randomly generate a fun fact about myself.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -2,17 +2,20 @@
...
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1 id="title">Lina's Portfolio</h1>
+      <p>Welcome to my portfolio! My name is Lina, and I'm a Latina in Tech currently interning at Google as a STEP intern. I am a rising Junior at Yale!</p>
```
Comment (KeatonTech) [inline portfolio/src/main/webapp/index.html]: nit: Try to wrap the lines so they fit in 100 columns (you can configure whatever text editor you're using to add a 'gutter' line at that column). Super long lines are harder to edit, especially on small screens

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -13,16 +13,16 @@
...
 /**
- * Adds a random greeting to the page.
+ * Adds a random fun fact to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFunFact() {
+  const funFacts =
+      ['I am a Capricorn sun, Aquarius Moon, and Cancer rising', 'I am fluent in Spanish and English', 'I adopted a dog when I was 15, and his name is Austin!', 'I am a first-generation college student.'];
```
Comment (KeatonTech) [inline portfolio/src/main/webapp/script.js]: similar nit: Put each fact on its own line. If you can set up TSLint in your editor I recommend it, that'll do some automatic formatting for you

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -2,8 +2,29 @@
   margin-left: auto;
   margin-right: auto;
   width: 650px;
+  color:lightslategray;
```
Comment (KeatonTech) [inline portfolio/src/main/webapp/style.css]: nit: Add a space after the colon, here and below

File: portfolio/src/main/webapp/style.css [hunk 4]
```diff
@@ -2,8 +2,29 @@
...
 }
 
 #greeting-container {
   margin-top: 20px;
 }
+
+img {
+  width: 25%;
+  height: 25%;
```
Comment (KeatonTech) [inline portfolio/src/main/webapp/style.css]: Does 'height: 25%' actually change anything here?

**Review by KeatonTech — CHANGES_REQUESTED** [review]

File: portfolio/src/main/webapp/style.css [hunk 5]
```diff
@@ -2,8 +2,29 @@
...
+  color:lightslategray;
+}
+
+body {
+  background-color:seashell;
+  font-family: 'Courier New', Courier, monospace;
+}
+
+button {
+  font-family: 'Courier New', Courier, monospace;
```
Comment (SarahGB) [inline portfolio/src/main/webapp/style.css]: Please alphabetize these, throughout this file.

File: portfolio/src/main/webapp/style.css (hunk 4 above)
Comment (linamontes10-zz) [reply portfolio/src/main/webapp/style.css]: I just realized it actually doesn't do anything - just got rid of that line!

File: portfolio/src/main/webapp/style.css [hunk 6]
```diff
@@ -1,4 +1,16 @@
...
+}
+
+button {
+  background-color: lightblue;
+  color: lightslategray;
+  font-family: 'Courier New', Courier, monospace;
+}
+
 #content {
+  color:lightslategray;
```
Comment (KeatonTech) [inline portfolio/src/main/webapp/style.css]: nit: Add a space after the colon

**Review by KeatonTech — APPROVED** [review]

**Pull request merged**